lib/ebaze_web/router.ex

```diff
@@ -17,6 +17,12 @@ defmodule EbazeWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+
+    resources "/users", UserController
+    
+    resources "/auctions", AuctionController
+
+    resources "/signin", SessionController, only: [:new, :create]
```

Comment:
`sign-in`?

Reply:
82464befda10fd9d75eb55e34eaefeafaa10469e